lib/resources/r4/document_reference.yaml

```diff
@@ -29,3 +94,140 @@ fields:
+  required: 'Yes'
+  type: Reference
+  action:
+    - create
+  description: Identifies who is responsible for adding the information to the document.
+  note: The author field is optional for system access.  For provider access, only a single author can be provided, and the author must be the authorized provider.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.author
+  example: |
+    {
+      "author": [
+        {
+          "reference": "Practitioner/123456"
+        }
+      ]
+    }
+
+- name: authenticator
+  required: 'No'
+  type: Reference
+  action:
+    - create
+  description: Which person or organization authenticates that this document is valid.
+  note: The authenticator field is only supported with system access. This field cannot be populated for provider access.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.authenticator
+  example: |
+    {
+      "authenticator": {
+        "reference": "Practitioner/123456"
+      }
+    }
+
+- name: content
+  required: 'Yes'
+  type: Attachment
+  action:
+    - create
+  description: The document and format referenced. There may be multiple content element repetitions, each with a different format.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.content
+  example: |
+   {
+      "content": [
+        {
+          "attachment": {
+            "contentType": "text/xml",
+            "url": "https://fhir-open.devcerner.com/r4/eb2384f8-839e-4c6e-8b29-18e71db1a0b1/Binary/R-17821258",
+            "title": "title",
+            "creation": "2020-07-29T21:02:04.000Z"
+          }
+        }
+      ]
+   }
+
+- name: content.attachment
+  required: 'Yes'
+  type: Attachment
+  action:
+    - create
+  description: The document or URL of the document along with critical metadata to prove content has integrity.
+  url: https://www.hl7.org/fhir/documentreference-definitions.html#DocumentReference.content.attachment
+  example: |
+   {
+     "attachment": {
+            "contentType": "text/xml",
+            "url": "https://fhir-open.devcerner.com/r4/eb2384f8-839e-4c6e-8b29-18e71db1a0b1/Binary/R-17821258",
+            "title": "title",
+            "creation": "2020-07-29T21:02:04.000Z"
+          }
+   }
+
+- name: context
+  required: 'Yes'
+  type: BackboneElement
+  action:
+    - create
+  description: The clinical context in which the document was prepared.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.context
+  example: |
+   {
+     "encounter": [
+       {
+         "reference": "Encounter/123456"
+       }
+     ],
+     "period": {
+       "start": "2020-01-01T00:00:00.000Z",
+       "end": "2020-01-01T01:00:00.000Z"
+     }
+   }
+
+- name: context.encounter
+  required: 'No'
+  type: Reference
+  action:
+    - create
+  description: Describes the clinical encounter or type of care that the document content is associated with.
```

Comment:
```suggestion
  description: Describes the clinical encounter that the document content is associated with.
  note: Only encounter references are supported
```

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/446/commits/eb5ca8d7a6905720c70e0c203d9de99739180394